mantidproject/mslice
# Eliminate pytest warnings

**Description of work:**
This PR eliminates all the pytest warnings seen when running pytest. It also ensures that pytest will fail whenever a new warning appears, so that we have a zero warning policy.

**To test:**
Code review
Make sure the tests are passing. Read the output to make sure there are no warnings

Fixes #884
